src/generate_container_host_image.py

```diff
@@ -61,7 +61,7 @@ def parse_args(zone, project):
 	# create dummy instance to build image in
 	try:
 		subprocess.check_call("""gcloud compute --project {proj} instances create {host} --zone {zone} \
-		  --machine-type n1-standard-1 --image ubuntu-minimal-1910-eoan-v20200107 \
+		  --machine-type n1-standard-1 --image ubuntu-minimal-2004-focal-v20210119a \
```

Comment:
Is this LTS? We don't want another case where Ubuntu pulls the rug out from under us.

Reply:
Yes, it's the LTS version. :-)